tools.py

```diff
@@ -0,0 +1,29 @@
+import getpass
+import requests
+import urllib
+import urlparse
+
+
+requests.packages.urllib3.disable_warnings()
+
+def vsphere_url(vm, host, args):
+    # Generates console URL for vSphere versions prior 6.0
```

Comment:
The docstring for the functions in this file do not match our docstring standards.